src/store/pollingMiddleware.js

```diff
@@ -0,0 +1,82 @@
+import { POLLING_IMAGES } from './action-types';
```

Comment:
This middleware is bound only to creating images. I'd try to make it a bit more disconnected to these cations, either rewrute this middleware so it creates any polled action, or let's use [MiddlewareListener](https://github.com/RedHatInsights/frontend-components/tree/master/packages/utils/src/MiddlewareListener), this way we don't really have to dispatch new actions and just spawn new query every few seconds (preferably trough `setTimeout`)